Apps/FlightSoftware/Watchdog2/src/ground_cmd.c

```diff
@@ -0,0 +1,498 @@
+/*
+ * ground_cmd.c
+ *
+ * Handle commands sent from the ground up to the watchdog
+ *
+ *  Created on: Nov. 29, 2020
+ *      Author: ohnx
+ */
+#include <stdint.h>
+#include "include/bsp.h"
+#include "include/buffer.h"
+#include "include/ip_udp.h"
+#include "include/uart.h"
+#include "include/adc.h"
+#include "include/flags.h"
+#include "include/i2c.h"
+
+extern uint8_t heatingControlEnabled, heaterStatus;
+extern uint16_t Kp_heater, heater_setpoint, heater_window, PWM_limit, heater_on_val, heater_off_val;
```

Comment:
I can definitely move these to main and have them passed to the heaterControl() function (and lump them into a struct), although I am not sure if these can be made as locals, since ground can send commands to us that change the values of each of these variables (see: handle_ground_cmd() in ground_cmd.c). I suppose I could pass a pointer to the struct to handle_ground_cmd()?